io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id": "io.gitlab.librewolf-community",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "20.08",
+            "add-ld-path": "."
+        }
+    },
+    "command": "librewolf",
+    "modules": [
+        "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        {
+            "name": "librewolf",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/lib",
+                "mv librewolf.tar.bz2 /app/lib/librewolf",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/usr/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/io.gitlab.librewolf-community.appdata.xml -t /app/share/metainfo",
+                "install -D -m644 io.gitlab.librewolf-community.desktop -t /app/share/applications",
+                "install -D -m755 librewolf.sh /app/bin/librewolf"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1274252735/artifacts/raw/LibreWolf-88.0.1-2.aarch64.tar.bz2",
+                    "sha256": "89584b80783c785bfa3539bfc269cdb653502ef06c8436419c04b9aa1eb78cd7",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "aarch64"
+                    ]
+                },
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1274251267/artifacts/raw/LibreWolf-88.0.1-2.x86_64.tar.bz2",
+                    "sha256": "dda65a5fc2c9d1e293bcf7ec82b755086a7e3dcf2d441b472de0f3ede9b5c046",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "io.gitlab.librewolf-community.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "librewolf.sh"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pcsc",
+        "--socket=pulseaudio",
+        "--persist=.librewolf",
+        "--persist=.mozilla",
```

Comment:
I understand why it's there but it's not something I'm happy about, especially when it's not 100% correct (for example, what about firefox-flatpak?)

Reply:
Hm… we might actually not need it (at least not anymore) – LibreWolf shouldn't have to touch `.mozilla` at all, except maybe for edge cases (plugins expecting `native-messaging-hosts` to be there), but those should probably be handled elsewhere (ie. on the plugin's end). I'd go with dropping it unless something breaks / something requires it.